appservice/src/index.ts

```diff
@@ -17,23 +14,27 @@ export * from './createAppService/AppServicePlanSkuStep';
 export * from './createAppService/CustomLocationListStep';
 export * from './createAppService/IAppServiceWizardContext';
 export * from './createAppService/LogAnalyticsCreateStep';
+export * from './createAppService/setLocationsTask';
+export * from './createAppService/SiteDomainNameLabelScopeStep';
 export * from './createAppService/SiteNameStep';
 export * from './createAppService/SiteOSStep';
-export * from './createAppService/setLocationsTask';
 export * from './createSlot';
 export * from './deleteSite/DeleteLastServicePlanStep';
 export * from './deleteSite/DeleteSiteStep';
 export * from './deleteSite/IDeleteSiteWizardContext';
-export * from './deploy/IDeployContext';
 export * from './deploy/deploy';
 export * from './deploy/getDeployFsPath';
 export * from './deploy/getDeployNode';
+export * from './deploy/IDeployContext';
 export * from './deploy/localGitDeploy';
-export { IPreDeployTaskResult, handleFailedPreDeployTask, runPreDeployTask, tryRunPreDeployTask } from './deploy/runDeployTask';
+export { handleFailedPreDeployTask, IPreDeployTaskResult, runPreDeployTask, tryRunPreDeployTask } from './deploy/runDeployTask';
 export * from './deploy/showDeployConfirmation';
 export { disconnectRepo } from './disconnectRepo';
 export * from './editScmType';
 export { registerAppServiceExtensionVariables } from './extensionVariables';
+export * from './KuduModels';
```

Comment:
Your linter reordered some stuff. I'm okay with it because I think mine tries to organize it the same way, but we should decide on a standard.